cro/cro.py

```diff
@@ -115,10 +113,16 @@ def broadcastspawning(self, REEF,REEFpob):
         spawners1 = REEFpob[spawners[0:int(nspawners/2)], :]
         spawners2 = REEFpob[spawners[int(nspawners/2):], :]
 
-        # get crossover mask for some of the methods below (one point crossover)
+        # get crossover mask for some of the methods below
         (a,b) = spawners1.shape
         mask = np.random.randint(2, size=[a,b])
-        mask = np.sort(mask, axis=0)
+        
+        # all zeros and all ones doesn't make sense, Not produces crossover
+        pos = np.where(np.sum(mask, axis= 1)==0)[0] 
```

Comment:
I would say these lines don't matter that much, i.e, if a line in `mask` were all 1's or all 0's then indeed there would be no crossover, but it would be due to the randomness of the whole process so it's not that bad in my opinion. In any case it does not hurt to check this.